modules/DataLaboratoryPlugin/src/main/resources/org/gephi/datalab/plugin/manipulators/columns/merge/Bundle.properties

```diff
@@ -4,7 +4,8 @@ JoinNumberColumns.name=Join numerical columns
 JoinNumberColumns.description=Join the values of number/number list columns into a new column with the type double[].
 CreateTimeInterval.name=Create time interval
 CreateTimeInterval.description=Create a time interval for each row taking 1 or 2 columns as start/end times.
-
+GenerateAnEdgeKind.name=Use the column to generate a typed edge
```

Comment:
Maybe we should leave the merge strategy out of this PR until it's ready, because it will create already resources for translators that are not a real feature yet, and that could be confusing.

Reply:
Ok fair enough I removed it